lib/freedom_patches/translate_accelerator.rb

```diff
@@ -122,7 +122,8 @@ def translate_no_override(key, options)
       dup_options = nil
       if options
         dup_options = options.dup
-        should_raise = dup_options.delete(:raise)
+        should_raise =
+          dup_options.delete(:raise) || Rails.application.config.i18n.raise_on_missing_translations
```

Comment:
A little bit of coupling here, but that's because our freedom patch isn't made to work with the built-in exception handler, which is how the Railtie makes this connection normally.